src/themeTemplates/themes.js

```diff
@@ -1,27 +1,29 @@
 import BootstrapSwitchButton from 'bootstrap-switch-button-react'
-import React, { useState } from 'react'
-import { useMediaQuery } from 'react-responsive'
+import React, { useState, useEffect } from 'react'
 
 const DARK_CLASS = 'dark'
 
 export const DarkToggle = () => {
-	const systemPrefersDark = useMediaQuery(
-		{
-			query: '(prefers-color-scheme: dark)',
-		},
-		undefined,
-		(prefersDark) => {
-			setIsDark(prefersDark)
+	const [isDark, setIsDark] = useState(false)
+	useEffect(() => {
+		if (localStorage.getItem('theme') === DARK_CLASS) {
+			document.documentElement.classList.add(DARK_CLASS)
+			setIsDark(true)
+		} else {
+			document.documentElement.classList.remove(DARK_CLASS)
+			setIsDark(false)
```

Comment:
can be written like this  -
const THEME = 'theme'

const isDarkThemeSelected = localStorage.getItem(THEME) === DARK_CLASS
setIsDark(isDarkThemeSelected)
if (isDarkThemeSelected) document.documentElement.classList.add(DARK_CLASS)
else document.documentElement.classList.remove(DARK_CLASS)